packages/overlays/stories/01-index.stories.mdx

```diff
@@ -0,0 +1,552 @@
+import { Story, Meta, html } from '@open-wc/demoing-storybook';
+import { renderLitAsNode } from '@lion/core';
+import {
+  OverlayMixin,
+  withBottomSheetConfig,
+  withDropdownConfig,
+  withModalDialogConfig,
+} from '../index.js';
+
+import './lion-demo-overlay.js';
+import './applyDemoOverlayStyles.js';1
+import { ref as r } from './directives/ref.js';
+
+<Meta title="Overlay System/Overview" parameters={{ component: 'lion-demo-overlay' }} />
+
+# Overlay System
+
+The overlay system allows to create different types of overlays like dialogs, toasts, tooltips,
+dropdown, etc.
+It's designed to be highly flexible, while still delivering sensible defaults.
+On top of this, the system was built having accessibility in mind.
+
+For a detailed rationale, please consult [Rationale](?path=/docs/overlay-rationale--page).
```

Comment:
Does this link work?

Reply:
yes - and in an upcoming storybook release it will not trigger a full page refresh 🙏